griptape/labs/tools/proxycurl/tool.py

```diff
@@ -0,0 +1,41 @@
+from griptape.artifacts import BaseArtifact, TextArtifact, ErrorArtifact
+from griptape.core import BaseTool
+from griptape.core.decorators import activity
+from schema import Schema, Literal
+from attr import define, field
+import requests
+
+
+@define
+class ProxycurlClient(BaseTool):
+    api_key: str = field(default=None, kw_only=True)
```

Comment:
Should be either `Optional[str]`, or no default `None` value, or a default empty string value.

Seems like making it optional makes sense and then `header_dic` could be conditional.

Reply:
@vasinov I think `api_key` should really just be a required field. Don't think the requests will work without it.